cmd/agent/app/integrations.go

```diff
@@ -40,6 +42,28 @@ var (
 	tufConfig    string
 )
 
+type versionStruct struct {
```

Comment:
you could name this struct `integrationVersion`/`wheelVersion` instead, to avoid the "struct" repetition